mistralrs-core/src/engine/mod.rs

```diff
@@ -347,6 +349,19 @@ impl Engine {
         }
     }
 
+    fn build_sequence_recognizer(constraint: &Constraint) -> SequenceRecognizer {
+        match constraint {
+            Constraint::Regex(rx) => {
+                SequenceRecognizer::Regex(StackRecognizer::from(RecRx::from_rx(rx)).into())
+            }
+            Constraint::Yacc(cfg) => SequenceRecognizer::Cfg(
+                // TODO: handle error
+                CfgParser::from_yacc(cfg).unwrap().into(),
```

Comment:
need to handle this

Reply:
This should be easy to handle, right? Can you please return a Result from this function?

Reply:
fixed it